test/ui-testing/pick-lists.js

```diff
@@ -0,0 +1,268 @@
+/* global describe, it, before, after, Nightmare */
+
+module.exports.test = (uiTestCtx) => {
+  const number = Math.round(Math.random() * 100000);
+  const editText = '-edited';
+  const pickList = `${number}pickList`;
+  const editedPickList = `${pickList}${editText}`;
+  const pickListValue = `${number}pickListValue`;
+  const editedPickListValue = `${pickListValue}${editText}`;
+
+  describe('Pick list crud', function test() {
+    const { config, helpers } = uiTestCtx;
+    const nightmare = new Nightmare(config.nightmare);
+    nightmare.options.width = 1300; // added this temporarily as MultiSelect doesnt work well with narrow screen sizes
+
+    this.timeout(Number(config.test_timeout));
+
+    describe('Login > Open settings > Create and edit pick list > Create and edit pick list value > Delete value and pick list > Logout\n', () => {
+      before((done) => {
+        helpers.login(nightmare, config, done);
+      });
+
+      after((done) => {
+        helpers.logout(nightmare, config, done);
+      });
+
+      it('should open Settings', done => {
+        helpers.clickSettings(nightmare, done);
+      });
+
+      it(`should create pick list ${pickList} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .waitUntilNetworkIdle(1000)
```

Comment:
These tests weren't reliably passing for me. Bumping all of these wUNI calls to a `waitUntilNetworkIdle(2000)` throughout got them to pass regularly.